server/load.js

```diff
@@ -134,7 +137,15 @@ exports.handleLoad = function(req, res, app, format) {
           filename += '/';
         }
 
-        var list = buildDirList(absfile, fs.readdirSync(absfile).sort());
+        var files = fs.readdirSync(absfile);
```

Comment:
So... consider:
   1. Moving the root user listing logic down here.
   2. Generalize getRootDirCache to getDirCache and handle both root and share.
   3. Maybe delete the buildDirList function entirely and use DirCache objects to load all directories.
   4. But probably don't actually cache most DirCache objects other than root and share.